cypress/e2e/notifications.spec.ts

```diff
@@ -12,7 +12,7 @@ import {
   visitE2eEmailTestRepo,
 } from "../utils"
 
-const UNREAD_COMMENTS_BG_COLOR = "rgb(235, 248, 255)"
+const UNREAD_COMMENTS_BG_COLOR = "rgb(248, 250, 254)"
```

Comment:
just to check - was these color changes from design system upgrade?

Reply:
I think so, @seaerchin can confirm

Reply:
![Screenshot 2023-08-08 at 1 57 22 PM](https://github.com/isomerpages/isomercms-frontend/assets/44049504/e3c1662b-8d10-4b74-a879-befdaaab767b)

`blue.50` token here got changed to isomer specific token